lbrynet/daemon/ExchangeRateManager.py

```diff
@@ -17,8 +17,10 @@
 
 class ExchangeRate(object):
     def __init__(self, market, spot, ts):
-        assert int(time.time()) - ts < 600
-        assert spot > 0
+        if not int(time.time()) - ts < 600:
+            raise AssertionError()
+        if not spot > 0:
+            raise AssertionError()
         self.currency_pair = (market[0:3], market[3:6])
```

Comment:
might as well make these of type ValueError  for clarity (first is because timestamp is too dated, second is because spot must be more than 0)